### DISCUSSION TITLE
Zim version 0.74.3 released !

### DISCUSSION THREAD

**Discussion by jaap-karssenberg** [zim-desktop-wiki/zim-desktop-wiki#1784] [Announcements]
Dear all,

Zim 0.74.3 has just been released. It contains a number of bug fixes and small enhancements.

Kind regards,

Jaap

##  0.74.3 - Tue 07 Dec 2021
* Fix issue with broken links on move page
* Fix bug in copy-paste
* Fix bug on move page for existing textbuffer
* Fix bug with saving of notebook editable state
* Fix for error while loading plugins
* Fix issue with stray unicode character
* Fix issue to allow relative document root
* Fix hack for rendering sourceview blocks
* Add mnemonic keys to open sidepane tabs
* Add links and screenshot to macOS app & update README information
* Improve inline rendering of equations & make image DPI confingurable
* Add box drawing characters to symbols list
* Scroll on moving lines with linesorter plugin
* Move floating table of contents to overlay layer

**Comment by dehesselle** [comment]
macOS app is being built right now and will be available in approximately two hours.

**Comment by jaap-karssenberg** [comment]
Hi Rene, I just see that the dev branch has failing tests after merging the
release. Could you run the test suite for the master as well on mac OS and
check for the same issue?  Probably it will give test errors, as it is not
up to date with dev on fixes we did on the test suite, but dev is crashing
with a segmentation fault, that is something else. If you see the same
issue, would need a "git bisect" to find what causes it as these
segmentation faults are not normal failures and python does not report what
went wrong where :(

**Comment by supertoothy** [comment]
Where may I find more information on Mnemonic Keys for opening sidepane tabs?

**Comment by jaap-karssenberg** [comment]
If you press "Alt" you should see underlined key values in the labels. Same
as menus.

Might not yet work in all translations

**Comment by ZtzEAcRsND** [comment]
Please upload zim-0.74.3.tar.gz to https://zim-wiki.org/downloads/

Thanks.

**Comment by jaap-karssenberg** [comment]
Done
